packages/openauth/src/issuer.ts

```diff
@@ -715,9 +715,7 @@ export function issuer<
   }
 
   function issuer(ctx: Context) {
-    const url = new URL(ctx.req.url)
-    const host = ctx.req.header("x-forwarded-host") ?? url.host
-    return url.protocol + "//" + host
+    return getRelativeUrl(ctx, "/")
```

Comment:
This causes problems with verification, because the issuer has a trailing slash, so the client tries to fetch from `https://myissuer.com//.well-known/jwks.json` which fails. I think we should remove the trailing slash.

Reply:
Here's what I switched it to in my own environment, which worked.

```suggestion
    return (new URL(getRelativeUrl(ctx, '/'))).origin
```